src/styles/price.scss

```diff
@@ -0,0 +1,19 @@
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  padding-left: $padding-left;
+
+  &--label {
+    color: $text-color;
+    font-weight: 400;
+    font-size: 12px;
+  }
+
+  &--value {
+    color: $text-color-strong;
```

Comment:
The variable `$text-color-strong` is used but not defined. Please define this variable to avoid compilation errors.